tools/mo/openvino/tools/mo/front/tf/GRUBlockCellReplacement.py

```diff
@@ -26,69 +26,52 @@ def find_and_replace_pattern(self, graph: Graph):
             tf_gru_block_cell.in_port(0).get_connection().set_destination(new_gru_cell.in_port(0))
```

Comment:
please sort out imports and clean-up

Reply:
Sorted out (but I've not found nothing to clean up).